templates/_components/card/card.html

```diff
@@ -1,7 +1,7 @@
 <section {% attrs class %} aria-labelledby="{{ title | slugify }}-title">
   <div class="bg-white shadow">
     {% if title or subtitle or button %}
-    <div class="flex flex-wrap gap-2 items-start justify-between px-4 py-2 sm:px-6 sm:py-4 sm:flex-nowrap">
+    <div class="flex flex-wrap gap-2 items-center justify-between px-4 py-2 sm:px-6 sm:py-4 sm:flex-nowrap">
```

Comment:
I don't see any practical difference between items-start and items-center here. What am I missing?